doc/user.rst

```diff
@@ -75,12 +75,12 @@ That points to the default user directory which can be changed by an environment
 
     MATRIXDEPOT_USERDIR=/...
 
-The data directory is found by::
+The data directory is queried by::
 
     julia> MatrixDepot.data_dir()
-    "/home/.../.julia/dev/MatrixDepot/data"
+    "/home/.../.julia/scratchspaces/MatrixDepot/data"
```

Comment:
Please cross-check the dir name. I think to remember it contains a randomized part.

Reply:
Good catch! I have updated the path to the correct UUID version.